data/map systems.txt

```diff
@@ -8738,6 +8741,46 @@ system Citadelle
 			distance 540.643
 			period 269
 
+system Clepsydra
+	pos 264 -874
+	attributes "bright star" "notable star"
+	arrival 750
+	habitable 750
```

Comment:
These numbers don't account for the presence of the `a-eater` star - the habitable range of this system would be 750 with just an A dwarf.
```suggestion
	arrival 4050
	habitable 4050
```
This suggestion puts the habitable distance from `a-eater` between that of a3 and a5.